examples/Subitems.ipynb

```diff
@@ -0,0 +1,67 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ipyleaflet\n",
+    "import json\n",
+    "import pandas as pd\n",
+    "from ipywidgets import link, FloatSlider\n",
+    "from branca.colormap import linear\n",
+    "import random\n",
+    "\n",
+    "center = (43, -100)\n",
+    "zoom = 4\n",
+    "geo_json_data = json.load(open(\"us-states.json\"))\n",
+    "m = ipyleaflet.Map(center=center, zoom=zoom)\n",
+    "unemployment = pd.read_csv(\"US_Unemployment_Oct2012.csv\")\n",
+    "unemployment = dict(\n",
+    "    zip(unemployment[\"State\"].tolist(), unemployment[\"Unemployment\"].tolist())\n",
+    ")\n",
+    "\n",
+    "legend = ipyleaflet.LegendControl(\n",
+    "    {\"low\": \"yellow\", \"medium\": \"orange\", \"high\": \"red\"},\n",
```

Comment:
I don't understand, this legend doesn't correspond to the colormap which is used in the choropleth layer (`linear.YlOrRd_04`)?